modules/gob-web/modules/course/revisions.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react'
+import styled from 'styled-components'
+import type {Course as CourseType} from '@gob/types'
+
+type Props = {
+	course: CourseType,
+}
+
+const RevisionTitle = styled.div`
+	padding-top: 16px;
+	font-weight: bold;
+`
+
+const Summary = styled.summary`
```

Comment:
I would rather you not remove the outline from the summary.

Reply:
Do you have a specific reason in mind?

Reply:
I think it's cleaner without it.

Reply:
Removed.